hpeOneView/resources/servers/id_pools_ipv4_subnets.py

```diff
@@ -23,18 +23,18 @@
 
 standard_library.install_aliases()
 
-from hpeOneView.resources.resource import ResourceClient
+from hpeOneView.resources.resource import Resource, ResourceSchemaMixin
 
 
-class IdPoolsIpv4Subnets(object):
+class IdPoolsIpv4Subnets(Resource, ResourceSchemaMixin):
     """
     The ID pools IPv4 subnets resource provides a Client API for managing IPv4 subnets.
     """
 
     URI = '/rest/id-pools/ipv4/subnets'
 
-    def __init__(self, con):
-        self._client = ResourceClient(con, self.URI)
+    def __init__(self, connection, data=None):
+        super(IdPoolsIpv4Subnets, self).__init__(connection, data)
 
     def create(self, resource, timeout=-1):
```

Comment:
no need to declare create method again since this method is already present in parent class. You need to re-define the method if in case you want to customize the existing method with some extra changes.

Remove create method.

Reply:
Done